packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -0,0 +1,193 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import '../lion-select-rich.js';
+
+describe('lion-select-rich', () => {
+  it('has an empty modelValue by default', async () => {
+    const el = await fixture(html`
+      <lion-select-rich>
+        <lion-listbox slot="input">
+          <lion-option value="nr1">Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el.modelValue).to.equal('');
+  });
+
+  it('has the selected option as modelValue', async () => {
+    const el = await fixture(html`
+      <lion-select-rich>
+        <lion-listbox slot="input">
+          <lion-option value="nr1" selected>Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el.modelValue).to.equal('nr1');
+  });
+
+  it('can preselect an option', async () => {
+    const el = await fixture(html`
+      <lion-select-rich .modelValue="${'nr2'}">
+        <lion-listbox slot="input">
+          <lion-option value="nr1">Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el._listboxElement.value).to.equal('nr2');
```

Comment:
Can you explain why the listboxElement has a `value` property equaling the selected value, instead of just `.modelValue`?

Reply:
Hey Joren, thanks for your questions: forces me to think through everything again and see if we got it right.

I’m writing a bigger explanation(should end up in docs) that addresses all of your questions (and similar questions), but basically it comes down to this:

- .modelValue is only needed on Field layer
- `<lion-listbox>` is equivalent to `<select>` and follows api
- slot="input" is _presentational_ from Application Developer perspective, it is the _interactive element_ from the Subclasser perspective
- combobox with autocomplete functionality has been taken into account, I can send you the design document on Wednesday
- apis are a compromise between accessibility/DX/performance and of course how they fit in the bigger picture (a.k.a. the form system)

For difference between Subclassers and Application Developers:  https://github.com/ing-bank/lion/blob/docs/shadowBem/packages/core/docs/definitions.md

The goal of every form web component we build is to end up as a FormControl. More specific, either as a Field or a Fieldset.
Please see https://github.com/ing-bank/lion/tree/master/packages/field and https://github.com/ing-bank/lion/blob/master/packages/field/docs/CustomFieldsTutorial.md.
Small recap: a FormControl provides a normalized api(including modelValue), validation, formatting, interaction states and an accessible html structure for labels, feedback, help texts and addons. 

Thorough explanation (docs) will follow ;)

Reply:
This is the more thorough explanation:  https://github.com/ing-bank/lion/blob/feat/richSelect/packages/select-rich/docs/DesignConsiderations.md

Hopefully addresses all questions you (and myself in the beginning, and probably consuming developers) have. Suggestions for naming/api improvements are welcome